server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -16,6 +16,14 @@ import           Hasura.RQL.Types.Common
 import           Hasura.RQL.Types.Permission
 import           Hasura.SQL.Types
 
+queryRootTy :: G.NamedType
+queryRootTy = G.NamedType "query_root"
+
+mutationRootTy :: G.NamedType
+mutationRootTy = G.NamedType "mutation_root"
+
+subscriptionRootTy :: G.NamedType
+subscriptionRootTy = G.NamedType "subscription_root"
```

Comment:
I think better naming would be `queryRootName`, `mutationRootName` etc.

Reply:
Since those functions give `G.NamedType` I name them as `.....Ty`. `...Name` is suitable if functions give `G.Name`.